make-web.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+
+readonly RELEASE_VERSION="v0.9.0-beta.3"
```

Comment:
should this come from an environment variable or are we going to manually update it before each release?